clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/AuthRefreshTests.scala

```diff
@@ -1,58 +0,0 @@
-package org.broadinstitute.clio.integrationtest.tests
-
-import java.time.temporal.ChronoUnit
-import java.time.{Instant, ZoneOffset, ZonedDateTime}
-
-import akka.stream.scaladsl.Source
-import com.google.auth.oauth2.TimeChangingChangeListener
-import org.broadinstitute.clio.client.commands.ClioCommand
-import org.broadinstitute.clio.integrationtest.EnvIntegrationSpec
-
-import scala.concurrent.duration._
-import scala.util.Success
-
-trait AuthRefreshTests { self: EnvIntegrationSpec =>
-
-  behavior of "Clio Client"
-
-  it should "refresh the auth token after it expires" in {
-
-    googleCredential.addChangeListener(
```

Comment:
All the shuffling around of auth-related code made it harder to access the `googleCredential` for this test. I had something nearly working, but something about the clock-shifting started causing the gcloud web services to return errors warning that the requester's clock wasn't accurate (which was true... just not sure why we haven't seen it before). Since the code on our side that this test was meant to cover is also [fully covered](https://codecov.io/gh/broadinstitute/clio/src/803f4087eab79376f9594531718d36412de245f6/clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/GoogleCredentialsGenerator.scala) by unit tests, I thought it'd be OK to just scrap this. I can try restoring it if others disagree.